runtime/router/router_test.go

```diff
@@ -47,19 +47,30 @@ func TestParamsFromContext(t *testing.T) {
 	r := &Router{}
 
 	handled := false
-	err := r.Handle("GET", "/:var",
-		http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-			params := ParamsFromContext(req.Context())
-			assert.Equal(t, 1, len(params))
-			assert.Equal(t, "var", params[0].Key)
-			assert.Equal(t, "foo", params[0].Value)
-			handled = true
-		}))
+	handlerFunc := http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		params := ParamsFromContext(req.Context())
+		assert.Equal(t, 1, len(params))
+		assert.Equal(t, "var", params[0].Key)
+		assert.Equal(t, "foo", params[0].Value)
+		handled = true
+	})
+	err := r.Handle("GET", "/:var", handlerFunc)
 	assert.NoError(t, err, "unexpected error")
 
 	req, _ := http.NewRequest("GET", "/foo", nil)
 	r.ServeHTTP(nil, req)
 	assert.True(t, handled)
+
+	r = &Router{}
+	r.WhitelistedPaths = []string{"/bar"}
```

Comment:
1. a test case to check without whitelist it doesn't allow the path these paths i.e. `/a/b` and `/a/:var`
2. a test case to check with whitelist it allows above.

Reply:
All those variety of test cases already exist in trie_test.go and trie_with_special_cases_test.go files

Reply:
Done